roles/matrix-bridge-mautrix-signal/templates/config.yaml.j2

```diff
@@ -152,15 +152,15 @@ bridge:
     # this to work. See https://github.com/tulir/mautrix-telegram/wiki/End‐to‐bridge-encryption
     encryption:
         # Allow encryption, work in group chat rooms with e2ee enabled
-        allow: false
+        allow: {{ matrix_mautrix_signal_enable_encryption }}
         # Default to encryption, force-enable encryption in all portals the bridge creates
         # This will cause the bridge bot to be in private chats for the encryption to work properly.
-        default: false
+        default: {{ matrix_mautrix_signal_enable_encryption }}
```

Comment:
This should use a separate variable (`matrix_mautrix_signal_bridge_encryption_default`).

For convenience, you can define it in terms of `matrix_mautrix_signal_bridge_encryption_allow`:

```yaml
matrix_mautrix_signal_bridge_encryption_default: "{{ matrix_mautrix_signal_bridge_encryption_allow }}"
```